server/models/validations/validate_required_properties_values.ts

```diff
@@ -1,22 +1,31 @@
 import { getInvEntityType } from '#controllers/entities/lib/get_entity_type'
 import { propertiesValuesConstraints as properties } from '#controllers/entities/lib/properties/properties_values_constraints'
+import { isInvPropertyUri } from '#lib/boolean_validations'
 import { newError } from '#lib/error/error'
-import type { Claims, PropertyUri } from '#types/entity'
+import { objectKeys } from '#lib/utils/types'
+import { isLocalEntityLayer } from '#models/entity'
+import type { InvEntity, PropertyUri } from '#server/types/entity'
 
-export function validateRequiredPropertiesValues (claims: Claims, checkedProperties?: PropertyUri[]) {
-  if (claims['wdt:P31']?.[0] == null) {
-    throw newError("wdt:P31 array can't be empty", 400, { claims })
+export function validateRequiredPropertiesValues (doc: InvEntity) {
+  const { claims } = doc
+  const isLocalLayer = isLocalEntityLayer(doc)
+  if (!isLocalLayer) {
+    if (claims['wdt:P31']?.[0] == null) {
+      throw newError("wdt:P31 array can't be empty", 400, { claims })
+    }
+    const type = getInvEntityType(claims['wdt:P31'])
+    if (validateControlledPropertiesClaimsPerType[type]) {
+      validateControlledPropertiesClaimsPerType[type](claims)
+    }
   }
 
-  const type = getInvEntityType(claims['wdt:P31'])
-  if (validateControlledPropertiesClaimsPerType[type]) {
-    validateControlledPropertiesClaimsPerType[type](claims)
-  }
-
-  checkedProperties = checkedProperties || (Object.keys(claims) as PropertyUri[])
-
-  checkedProperties.forEach(property => {
+  objectKeys(claims).forEach(property => {
     validateUniqueValue(property, claims[property])
+    if (isLocalLayer) {
+      rejectRemotePropertiesOnLocalLayer(property, doc)
```

Comment:
I understand `reject` as the opposite of `filter`, but this naming actually infers `reject` as the opposite of `validate`. I would be in favour of only using the verb `validate` for such functions.

```suggestion
      validateLocalLayerWithoutRemoteProperties(property, doc)
```

Reply:
Done in 70fda245c